lib/beaker/test_suite.rb

```diff
@@ -172,6 +172,7 @@ def write_junit_xml(xml_file)
             end
             suite.add_child(properties)
 
+            @test_cases.sort! { |x,y| y.runtime <=> x.runtime } if time_sort
```

Comment:
I am vaguely concerned about an in-place sort for global attribute that may have other uses in the future that are affected by the order.

Also, it's not clear from the `write_junit_xml` method name or the `time_sort` parameter that it will modify the `test_cases` ordering.